### DISCUSSION TITLE
Nested dropdowns extend beyond screen space

### DISCUSSION THREAD

**Discussion by uzervlad** [ppy/osu#15599] [Q&A] [answered]
Couldn't find an issue about this, but I'll send it here first, just in case.

![osu_2021-11-12_23-07-27](https://user-images.githubusercontent.com/24654479/141528740-4849f8ba-e0ec-4a04-ba46-7f0faa886971.jpg)

![osu_2021-11-12_23-11-26](https://user-images.githubusercontent.com/24654479/141528711-901d40b7-c8f3-4e49-94c1-edefb8689e35.jpg)

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu-framework/issues/3861
